ci/precise_benchmarks.py

```diff
@@ -0,0 +1,150 @@
+import subprocess
+import os
+import sys
+import json
+from time import gmtime, strftime
+import platform
+import git
```

Comment:
Blocking: This script won't work unless the user installs extra packages. List the `git` dependency in a `requirements.txt` file.